v2.2/follower-reads.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Follower Reads
+summary: To reduce latency for read queries, you can choose to have the closest node serve the request using the "Follower reads" feature.
+toc: true
+---
+
+<span class="version-tag">New in v2.2:</span> To reduce latency for read queries, you can choose to have the closest node serve the request using the follower read  feature. For this to work, though, the response might not have the most recent version of the data, though the data will be consistent at the point in time the read is served from.
+
+{{site.data.alerts.callout_danger}}
+The Follower Read feature is only available to [enterprise](https://www.cockroachlabs.com/product/cockroachdb/) users.
+{{site.data.alerts.end}}
+
+## What are Follower reads?
+
+Follower reads are a mechanism to let any replica of a range serve a read request, but are only available for read queries that are sufficiently in the past, i.e., using `AS OF SYSTEM TIME`. Currently, follower reads are available for any read operation at least 48 seconds in the past, though there is active work reduce that window. 
+
+To future-proof this feature in your code, we've also included a convenience function that runs your queries at a time as close as possible to the present time while remaining safe for follower reads.
```

Comment:
> To future-proof this feature in your code, we've also included a convenience function that runs your queries at a time as close as possible to the present time while remaining safe for follower reads.

> In widely distributed deployments, using follower reads can reduce the latency of read operations (which can also increase throughput) by letting the replica closest to the gateway serve the request, instead of forcing the gateway to communicate with the leaseholder, which could be geographically distant.

I think the order of these two paras should be reversed, on the argument that the info in the current 3rd para is more important/general than the 2nd.